new-era-commands/slash/searchmdn.js

```diff
@@ -0,0 +1,28 @@
+const { SlashCommandBuilder, EmbedBuilder } = require('discord.js');
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('searchmdn')
+    .setDescription('Search for terms in MDN')
+    .addStringOption((option) => option
+      .setName('prompt')
+      .setDescription('term to search')
+      .setRequired(true))
+    .addUserOption((option) => option.setName('user').setDescription('user to ping')),
+  execute: async (interaction) => {
+    const userId = interaction.options.getUser('user')?.id;
+    const prompt = interaction.options.getString('prompt');
+    const searchUrl = `https://developer.mozilla.org/en-US/search?q=${prompt.replaceAll(' ', '+')}`;
+
+    const searchEmbed = new EmbedBuilder()
+      .setColor('#cc9543')
+      .setTitle('Search MDN')
+      .setDescription(
+        `Here are the [MDN results for "${prompt}"](${searchUrl})`,
+      );
+    await interaction.reply({
```

Comment:
I don't think we want to "reply" to the user that entered the command most of the time? Or maybe it could be a use-case specific to the slash command, a way to get a link?

Reply:
This is the same behavior as with all other slash commands:

![image](https://github.com/TheOdinProject/odin-bot-v2/assets/76259120/cac69a89-0c2d-40af-9504-d6d8b380286a)

Reply:
Oh yeah true 👍